src/fheroes2/dialog/dialog_audio.cpp

```diff
@@ -146,20 +135,22 @@ namespace Dialog
 
         drawDialog( roi );
 
-        const fheroes2::Point buttonOffset( 112 + dialogArea.x, 252 + dialogArea.y );
-        fheroes2::Button buttonOkay( buttonOffset.x, buttonOffset.y, isEvilInterface ? ICN::BUTTON_SMALL_OKAY_EVIL : ICN::BUTTON_SMALL_OKAY_GOOD, 0, 1 );
+        Settings & conf = Settings::Get();
+        const bool isEvilInterface = conf.isEvilInterfaceEnabled();
 
-        buttonOkay.draw();
+        fheroes2::Button buttonOk;
+        const int buttonOkIcnId = isEvilInterface ? ICN::BUTTON_SMALL_OKAY_EVIL : ICN::BUTTON_SMALL_OKAY_GOOD;
+        background.renderButton( buttonOk, buttonOkIcnId, 0, 1, { 0, 11 }, fheroes2::StandardWindow::Padding::BOTTOM_CENTER );
 
         display.render();
```

Comment:
Since we are doing the changes in the nearby code lets optimize the rendering a bit. :)
```suggestion
        display.render( background.totalArea() );
```